src/main/java/net/darkhax/bookshelf/lib/util/HolidayUtils.java

```diff
@@ -0,0 +1,63 @@
+package net.darkhax.bookshelf.lib.util;
+
+import net.darkhax.bookshelf.lib.Constants;
+
+import java.util.Calendar;
+
+public class HolidayUtils {
+    public static Boolean isNewYears = false;
+    public static Boolean isAprilFirst = false;
+    public static Boolean isHalloween = false;
+    public static Boolean isChristmas = false;
+
+    public static void checkDates () {
+
+        dateChecker();
+        if (isNewYears == true) {
```

Comment:
Remove `== true` and maybe change it to if/else, it can't be two holidays at once